test/k8s-integration/driver-config.go

```diff
@@ -29,7 +29,7 @@ const (
 
 	// configurable timeouts for the k8s e2e testsuites.
 	podStartTimeout       = "600s"
-	claimProvisionTimeout = "600s"
+	claimProvisionTimeout = "1800s"
```

Comment:
lets use timeouts per tier. large timeout can hide issues in standard and premium tiers

Reply:
done